doc/HOWTO_RELEASE.rst.txt

```diff
@@ -49,7 +49,7 @@ macports - see this `OSX wheel building summary
 
 Windows
 -------
-We build 32- and 64-bit wheels on Windows. Windows 7, 8 and 10 are supported.
+We build 32-bit and 64-bit wheels on Windows. Windows 7, 8 and 10 are supported.
```

Comment:
Probably not worth much energy to debate, but this usage is pretty common per i.e., https://english.stackexchange.com/a/39862